numpy-stubs/__init__.pyi

```diff
@@ -450,6 +451,11 @@ def array(
     subok: bool = ...,
     ndmin: int = ...,
 ) -> ndarray: ...
+def asarray(
+    a: Union[SupportsFloat, Sequence, ndarray],
+    dtype: _DtypeLike = ...,
```

Comment:
Should this be optional?

Reply:
Ah, yes. Good catch!

(I think I got a bit confused w.r.t. the stub format for saying that something has a default value. Will make the change.)